packages/link/src/Link.tsx

```
@@ -6,8 +6,10 @@ import { useLinkComponent } from './linkContext';
 
 export type LinkProps = LinkComponentProps;
 
-export const Link = forwardRefWithAs<'a', LinkProps>((props, ref) => {
+export const Link = forwardRefWithAs<'a', LinkProps>(({ href, data }, ref) => {
```

I reckon this component might be OK to keep to consumer prop spreading. It's just an un-styled link component, without the extra props it's reasonably useless!

---

Also `LinkProps` has all the valid props for an anchor element, so you can pass `Link` a bunch of props, but it won't do anything with them so it fails silently.
e.g. in the deploy preview for this PR the Brighte and GitHub logos aren't styled since the `className` prop isn't getting forwarded on.

---

Yeah was tweaking with this on the preview branch, was hoping I could get it working without the spread but think you're right